kdesk/icon-hooks.sh

```diff
@@ -82,7 +82,9 @@ case $icon_name in
     else
         num_file=""
     fi
-    printf "IconStatus: {190,53} $num_file"
+    if [ "$num_file" != "" ]; then
+	printf "IconStatus: {190,53} $num_file\n"
+    fi
```

Comment:
@skarbat  Will this cause a problem because `num_file` is empty or could it also cause a problem if the line is missing completely?